scheduler/src/cook/mesos/task.clj

```diff
@@ -14,43 +14,41 @@
 ;; limitations under the License.
 ;;
 (ns cook.mesos.task
-  (:require [clojure.tools.logging :as log]
-            [cook.mesos.util :as util]
+  (:require [cook.mesos.util :as util]
             [plumbing.core :refer (map-vals)])
   (:import com.netflix.fenzo.TaskAssignmentResult))
 
-(defonce custom-executor-name "cook_agent_executor")
-(defonce custom-executor-source "cook_scheduler")
+(def custom-executor-name "cook_agent_executor")
+(def executor-source "cook_scheduler")
```

Comment:
I didn't find these being defined elsewhere. Are they were not intentionally set to `defonce`?

I'll add the `defonce` back.